deploy/roles/python-libs7/tasks/main.yml

```diff
@@ -55,6 +55,7 @@
     name: "{{ python3_packages }}"
     executable: pip3
     umask: '0022'
+    extra_args: --ignore-installed
```

Comment:
I will try. The issue was with uninstall of older version of the PyYAML package, not with installing it. It installed version 5.4.1 and failed on uninstalling version 3.13:
Collecting PyYAML\n  Using cached PyYAML-5.4.1-cp36-cp36m-manylinux1_x86_64.whl (640 kB) 
....
Found existing installation: PyYAML 3.13\n\n:stderr:     ERROR: Command errored out with exit status 1:\n     command: /usr/bin/python3 -c 'import io, os, sys, setuptools, tokenize;  .......
Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\nERROR: Cannot uninstall 'PyYAML'. It is a distutils installed project and thus we cannot accurately determine which files belong to it which would lead to only a partial uninstall.\n"}